runtime/decimal.c

```diff
@@ -96,6 +104,9 @@ TaggedPtrPanicCode finish(decQuad *dq, decContext *cx) {
             result.panicCode = 0;
             result.ptr = createDecimal(dq);
         }
+        else if (status & DEC_Division_impossible) {
```

Comment:
Why can't we just use PANIC_ARITHMETIC_OVERFLOW for this?

Reply:
I used the error given by the library, yes we can use PANIC_ARITHMETIC_OVERFLOW, because DEC_Division_impossible happens when the operations have too many digits.
Changed.

Reply:
Relevant tests are added from L:162 to L:168 in `testRem()` function.